herbert_core/utilities/files/do_fseek.m

```diff
@@ -0,0 +1,16 @@
+function status = do_fseek(fid, offset, origin)
+%%DO_FSEEK Call fseek with the given arguments, throw an error if something goes wrong
```

Comment:
`fseek` does not thrown in Matlab, so `try/catch` is in mainly useless.  A lot of unnecessary changes to extract this, but even more unnecessary efforts to revert

Reply:
Afraid it does:
```
>> q = fopen('dumpy', 'w')
>> try
fseek(q, -1000, 'bot')
catch ME
ME
end

ME =

  MException with properties:

    identifier: 'MATLAB:badorigin_mx'
       message: 'Invalid origin.'
         cause: {}
         stack: [0x1 struct]
    Correction: []

>> fseek(7, 1, 'bof')
Error using fseek
Invalid file identifier.  Use fopen to generate a valid file identifier.

```

Reply:
OK then. Either my knowledge is from very old Matlab version or I used it in the situations, where input parameters were verified in different place. As separate routine its necessary